### PR TITLE
Android blog post

### PR DESCRIPTION
**Pull request by paulrouget**
Fix https://github.com/servo/servo.org/issues/63

Depends on https://github.com/servo/download.servo.org/pull/41

Do not merge yet. Waiting on more details about https://github.com/servo/servo/issues/21317

---
This change is [Reviewable](https://reviewable.io/reviews/servo/blog.servo.org/164)

### CODE REVIEW

**Comment by paulrouget** [comment]
@jdm can you look proofread this ?

File: _posts/2018-08-22-android-update.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+layout:     post
+title:      Servo for Android: nightly builds available and contribution opportunities
+date:       2018-08-22 09:00:00
+summary:    Servo for Android: nightly builds available and contribution opportunities
+categories:
+---
+
+We [recently revamped](https://github.com/servo/servo/pull/20912) Servo on Android.
```
Comment (larsbergstrom) [inline _posts/2018-08-22-android-update.md]: Maybe something about the motivation/why? Something like "As we continue to work on new implementation techniques and support for new platforms and technologies, we'd like to make this build available for testing and feedback."

File: _posts/2018-08-22-android-update.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+We provide [nightly builds of an Android Servo view](https://download.servo.org/nightly/android/servo-latest.aar) (aar). The API is minimal and pretty straight forward. If you are interested in testing Servo in your Android project, see the [Servo application](https://github.com/servo/servo/tree/master/support/android/apk/servoapp) as an easy example to follow.
+
+It's also a lot easier (and faster) to build Servo for Android now. See [instructions here](https://github.com/servo/servo/wiki/Building-for-Android#working-on-the-user-interface-without-building-servo).
+
+<img width="300" alt="Android Studio Screenshot" src="https://user-images.githubusercontent.com/373579/40770306-e90e6c24-64ec-11e8-820f-306feb9512e7.png">
+
+## How you can help
+
+There are many low hanging fruits. We need help filing and fixing issues.
```
Comment (larsbergstrom) [inline _posts/2018-08-22-android-update.md]: "There is a lot of low hanging fruit."?

File: _posts/2018-08-22-android-update.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+layout:     post
+title:      Servo for Android: nightly builds available and contribution opportunities
+date:       2018-08-22 09:00:00
+summary:    Servo for Android: nightly builds available and contribution opportunities
+categories:
+---
+
+We [recently revamped](https://github.com/servo/servo/pull/20912) Servo on Android.
+
```
Comment (larsbergstrom) [inline _posts/2018-08-22-android-update.md]: Is there a preview you could put here that shows something compelling? Either a quick video/gif of the product working or of a side-by-side of page scroll against a competing browser on the same platform? Something that will help get people excited, above and beyond the APK availability (which is totally awesome, btw!)

Comment (paulrouget) [reply _posts/2018-08-22-android-update.md]: There are screenshots. As for getting people excited, we are not delivering much in term of user experience. There's not much to show.

**Comment by paulrouget** [comment]
@larsbergstrom (my comment was marked as outdated):

> Is there a preview you could put here that shows something compelling? Either a quick video/gif of the product working or of a side-by-side of page scroll against a competing browser on the same platform? Something that will help get people excited, above and beyond the APK availability (which is totally awesome, btw!)

There are screenshots. As for getting people excited, we are not delivering much in term of user experience. There's not much to show.

**Comment by larsbergstrom** [comment]
Thanks, @paulrouget! I see the screenshots now. I was thinking more of whether we had anything like the old-school https://twitter.com/jaffathecake/status/705355715414781953, but updated for mobile instead of desktop browsers, not something featuring a snazzy new UX interaction. Totally P2/nice to have and something we can follow up with later.

**Comment by paulrouget** [comment]
The demo doesn't work: https://github.com/servo/servo/issues/21505

**Comment by mrego** [comment]
Moved to https://github.com/servo/servo.org/pull/71

**Pull request closed without merging**